test/unit/whitehall/asset_manager_storage_test.rb

```diff
@@ -70,4 +70,12 @@ class Whitehall::AssetManagerStorage::FileTest < ActiveSupport::TestCase
 
     file.delete
   end
+
+  test 'uses the asset-manager api to return the url of the file' do
+    asset_url = 'http://asset-host/asset.png'
+    Services.asset_manager.stubs(:whitehall_asset).with('/government/uploads/path/to/asset.png').returns('file_url' => asset_url)
+
+    file = Whitehall::AssetManagerStorage::File.new('path/to/asset.png')
```

Comment:
Minor: Consider extracting this into a common setup block.

Reply:
I've added another commit to extract some of the common code into a `setup` block.